### PR TITLE
branch 1: Test20.java push

### PR DESCRIPTION
**Pull request by gitKapoya**
yoyoyo! check, please

### CODE REVIEW

File: src/com/gitfreeit/kapoya/branch1/Task20.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        System.out.println(s);
+        s = trimString(s);
+        System.out.println("\nТекст, составленный из последних букв всех слов: ");
+        eachLastLetter(s);
+    }
+
+    // удаляет лишние символы и пробелы
+    public static String trimString(String s) {
+        s = s.replaceAll("[~@#$%^&*()_+=,\"»«:;.?!0123456789]", "");
+
```
Comment (mrnchernova) [inline src/com/gitfreeit/kapoya/branch1/Task20.java:17]: have to use regex

Comment (mrnchernova) [reply src/com/gitfreeit/kapoya/branch1/Task20.java:17]: have to use regex 2

Comment (gitKapoya) [reply src/com/gitfreeit/kapoya/branch1/Task20.java:17]: lalala

**Review by gitKapoya — COMMENTED** [review]
add Task20_2 for check

**Pull request merged**